File: codeup/100000569/E.py
#!/usr/bin/python3

str1 = input()
str2 = input()

len1 = len(str1)
len2 = len(str2)
i = 0
while i < len1 and i < len2:
    if str1[i] < str2[i]:
        print(ord(str1[i])-ord(str2[i]))
        break
    elif str1[i] > str2[i]:
        print(ord(str1[i])-ord(str2[i]))
        break
    i += 1

if i == len1 or i == len2:
    print(0)

File: codeup/100000569/E.cpp
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main() {
  char str1[100], str2[100];
  fgets(str1, 100, stdin);
  fgets(str2, 100, stdin);

  int i = 0;
  for (; str1[i] != '\0' && str2[i] != '\0'; i++) {
    if (str1[i] < str2[i]) {
      printf("%d\n", str1[i] - str2[i]);
      break;
    } else if (str1[i] > str2[i]) {
      printf("%d\n", str1[i] - str2[i]);
      break;
    }
  }
  if (str1[i] == '\0' || str2[i] == '\0') printf("0\n");
  return 0;
}